.circleci/config.yml

```diff
@@ -6,15 +6,53 @@ commands:
       - restore_cache:
           name: Restore Yarn Cache
           keys:
-            - yarn-{{ checksum "yarn.lock" }}
+            - yarn-v2-{{ arch }}-{{ checksum "yarn.lock" }}
       - run:
           name: Yarn Install
-          command: yarn install --frozen-lockfile
+          command: YARN_CACHE_FOLDER=~/.cache/yarn yarn install --frozen-lockfile
       - save_cache:
           name: Save Yarn Cache
-          key: yarn-{{ checksum "yarn.lock" }}
+          key: yarn-v2-{{ arch }}-{{ checksum "yarn.lock" }}
           paths:
             - ~/.cache/yarn
+  react-native-install:
+    steps:
+    - run:
+        name: Install command line tools
+        command: |
+          sudo npm install -g react-native-cli
+  build-android:
+    steps:
+      - run:
+          name: Change initial HTML file
+          command: |
+            cp ./bin/tmp/initial-device-tests-html.js ./src/app/initial-html.js
+      - run:
+          name: Bundle Debug android
+          command: |
+            mkdir -p android/app/src/main/assets
+            react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res
```

Comment:
Probably worth creating an npm script for that, we could call it `bundle:android:test` for instance

Reply:
Hmm yeah it's probably worth making that update. I'll make that update.